testutil/network/util.go

```diff
@@ -78,6 +78,15 @@ func startInProcess(cfg Config, val *Validator) error {
 		}
 	}
 
+	if val.AppConfig.GRPC.Enable {
```

Comment:
It would be nice to mention the relevant PR, cosmos/cosmos-sdk#14652, in the PR description.